refresh-example.sh

```diff
@@ -37,9 +37,11 @@ fi
 \rm -fr example
 
 # Make the new example
-npx react-native init example
+npm_config_yes=true npx react-native init example --skip-install
 pushd example
-npx react-native-windows-init --overwrite
+rm -f _ruby-version Gemfile*
```

Comment:
if you don't do this, and you are not running the *exact* version of ruby that react-native template wants, it fails the install, it's really annoying and I'm going to work to change it upstream. I use ruby 3.1.0 locally not 2.7.4 and it should work (I know because I worked to fix all the incompatibilities!).
So just delete the ruby version pinning stuff.